xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml

```diff
@@ -67,9 +67,9 @@
         <ReturnType>System.Collections.Generic.IReadOnlyList&lt;T&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the explicit bucket boundaries advised to be used with histogram instruments.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>&lt;para&gt;Notes:&lt;/para&gt;&lt;para&gt;&lt;list type="bullet"&gt;&lt;/para&gt;&lt;para&gt;&lt;item&gt;A &lt;see langword="null" /&gt; value means no bucket boundaries have been configured and default values should be used for bucket configuration.&lt;/item&gt;&lt;/para&gt;&lt;para&gt;&lt;item&gt;An empty set of bucket boundaries hints that the histogram by default should NOT contain buckets and should only track count and sum values.&lt;/item&gt;&lt;/para&gt;&lt;para&gt;&lt;item&gt;A set of distinct increasing values for bucket boundaries hints that the histogram should use those for its default bucket configuration.&lt;/item&gt;&lt;/para&gt;&lt;para&gt;&lt;/list&gt;&lt;/para&gt;</remarks>
```

Comment:
This looks wrong:

![image](https://github.com/user-attachments/assets/4738cf2f-1bdc-4a9c-9adc-da8707e086f5)

Reply:
I don't know what happened there 😖

Reply:
Looks the tool cannot handle porting such format?

https://github.com/dotnet/runtime/blob/62784822a187d734cc13b595aaadc315e1c915e0/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/InstrumentAdvice.cs#L28